pallets/logion_loc/src/tests.rs

```diff
@@ -22,11 +22,56 @@ fn it_creates_loc() {
 			files: vec![],
 			closed: false,
 			loc_type: LocType::Transaction,
-			links: vec![]
+			links: vec![],
+			void_info: None,
+			replacer_of: None
 		}));
 	});
 }
 
+#[test]
+fn it_makes_existing_loc_void() {
+	new_test_ext().execute_with(|| {
+		assert_ok!(LogionLoc::create_loc(Origin::signed(LOC_OWNER1), LOC_ID, LOC_REQUESTER, LocType::Transaction));
+		assert_ok!(LogionLoc::make_void(Origin::signed(LOC_OWNER1), LOC_ID, "reason".as_bytes().to_vec(), None));
+
+		let void_info = LogionLoc::loc(LOC_ID).unwrap().void_info;
+		assert!(void_info.is_some());
+		match void_info.unwrap() {
+			LocVoidInfo::V1 { reason: _reason, replacer: _replacer } => {
+				assert_eq!(_reason, "reason".as_bytes().to_vec());
+				assert_eq!(_replacer, None)
+			}
```

Comment:
What happens when there is no match?

Reply:
There is indeed a way to provide a default branch, but the compiler complains that it(s unreachable.